smartmin/models.py

```diff
@@ -274,6 +290,7 @@ def unicode_csv_reader(utf8_data, dialect=csv.excel, **kwargs):
 
         records = []
         num_errors = 0
+        error_description = dict()
```

Comment:
Another way would be to use a list of dicts here for the errors.. since I think this all ends up being JSONified that's a more natural representation that maintains ordering.